Resources/doc/internals/configuring_the_http_client.md

```diff
@@ -27,4 +27,93 @@ hwi_oauth:
 
 If you don't want to use the HTTPlug Bundle, the client and factory services **must be specified**.
 
+## Difference since 0.6+ version
+
+On v0.5 and below, you was able to configure your http client like that:
+
+```yaml
+# app/config/config.yml
+
+hwi_oauth:
+    http_client:
+        timeout:       10 # Time in seconds, after library will shutdown request, by default: 5
```

Comment:
Time in seconds after which pending HTTP requests are cancelled

Reply:
This is a copy/paste of the old doc, not related. I won't touch it here.